README.md

```diff
@@ -115,46 +141,55 @@ Debian packages will be generated in `images` folder.
 <summary>JP6.0 build results</summary>
 
 - kernel image (not modified): `images/6.0/rootfs/boot/Image`
+- dtb: `images/6.0/rootfs/boot/dtb/tegra234-p3737-0000+p3701-0000-nv.dtb`
 - dtb overlay: `images/6.0/rootfs/boot/tegra234-camera-d4xx-overlay.dtbo`
-- oot modules: `images/6.0/rootfs/lib/modules/5.15.122-tegra/extra`
+- oot modules: `images/6.0/rootfs/lib/modules/5.15.136-tegra/updates`
 
 Following steps required:
 
-1.	Copy entire directory `images/6.0/rootfs/lib/modules/5.15.122-tegra/extra/` from host to `/lib/modules/5.15.122-tegra/extra/` on Orin
-2.	Copy `tegra234-camera-d4xx-overlay.dtbo` from host to `/boot/tegra234-camera-d4xx-overlay.dtbo` on Orin
-3.	Run  $ `sudo /opt/nvidia/jetson-io/jetson-io.py`
+1.	Copy entire directory `images/6.0/rootfs/lib/modules/5.15.136-tegra/updates` from host to `/lib/modules/5.15.136-tegra/` on Orin
+2.	Copy entire directory `images/6.0/rootfs/lib/modules/5.15.136-tegra/extra` from host to `/lib/modules/5.15.136-tegra/` on Orin
+3.	Copy `tegra234-camera-d4xx-overlay.dtbo` from host to `/boot/tegra234-camera-d4xx-overlay.dtbo` on Orin
+4.  Copy `tegra234-p3737-0000+p3701-0000-nv.dtb` from host to `/boot/dtb/` on Orin
+5.	Run  $ `sudo /opt/nvidia/jetson-io/jetson-io.py`
```

Comment:
this should be done before p.4, otherwise jetson-io.py will not run.

Reply:
OK, let's fix the readme and merge it